docker-compose.yaml

```diff
@@ -25,6 +25,10 @@ services:
       - REDIS_HOST=redis
       - REDIS_PORT=6379
       - DELTALAKE_WAREHOUSE_DIR=s3a://cdm-lake/warehouse
+      - SPARK_EVENT_LOG_DIR=s3a://cdm-spark-events/spark-events
```

Comment:
I realize this is just an example setup, but could we name this something else? It seems easily confused with https://github.com/kbase/cdm-spark-events